test/util/timer.cpp

```diff
@@ -43,7 +43,11 @@ using namespace realm::test_util;
 
 uint_fast64_t Timer::get_timer_ticks() const
 {
+#if !REALM_UWP
```

Comment:
It seems this at the very least requires a FIXME, because benchmarks and other timing-sensitive code is obviously going to be totally broken.